editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4349,6 +4349,20 @@ void CanvasItemEditor::_popup_callback(int p_op) {
 			}
 
 		} break;
+		case CLEAR_GUIDES: {
+			Array hguides = EditorNode::get_singleton()->get_edited_scene()->get_meta("_edit_horizontal_guides_");
```

Comment:
You should likely check `has_meta` before getting it, otherwise this will raise an error in the terminal if you don't have any guides.

Reply:
this begs the question why there's no get_meta with default like in the case of Dictionary.get(key, default_value) 🤔 

I'll change it!